doc/nrf/includes/sample_snippets.txt

```diff
@@ -3,3 +3,9 @@ You can find the snippets in the :file:`snippets` directory of the sample.
 
 Specify the corresponding snippet names in the |SNIPPET| CMake option.
 For more information about using snippets, see :ref:`zephyr:using-snippets` in the Zephyr documentation.
+
+  .. note::
+    This snippet requires a custom ZAP configuration, located in the :file:`snippets/schedules` directory.
+    Ensure to configure and generate the correct ZAP files if a custom data model is needed.
+    To do this, run the ``west zap-generate`` command and choose the :file:`snippets/schedules/lock.zap` file.
+
```

Comment:
```suggestion
```
I don't think the note is supposed to go to the Thread samples using this include.

Reply:
Oh, that's right... I added it here before creating the matter_snippets_note.txt file and forgot to remove it :/